pkg/resourcetopologyexporter/resourcetopologyexporter.go

```diff
@@ -56,19 +58,28 @@ func ContainerIdentFromString(ident string) (*podrescli.ContainerIdent, error) {
 	return cntIdent, nil
 }
 
+func Execute(nrtupdaterArgs nrtupdater.Args, resourcemonitorArgs resourcemonitor.Args, rteArgs Args) error {
```

Comment:
We are no longer using Execute() and are now using the ExecuteWithPodresourcesClient() instead. Has this been left here intentionally or it is a mistake?

Reply:
It was intentional, but on hindsight it makes no real sense. Let me rename the latter into `Execute`